bigquery_etl/cli/check.py

```diff
@@ -53,7 +55,7 @@ def _render_result_split_by_marker(marker, rendered_result):
 
     for sql_statement in rendered_result:
         sql_statement = sql_statement.strip()
-        if re.search(f"^#{marker}", sql_statement, re.IGNORECASE):
+        if re.search(f"#{marker}", sql_statement, re.IGNORECASE):
```

Comment:
We could remove this change. But then we'd have to move the `#fail` marker to above the `{{ header }}`, which feels strange for a header.

In reality I don't see why the marker needs to be at the beginning, but happy to change this back if the team thinks otherwise.